Sources/XcodeGraph/Models/BuildAction.swift

```diff
@@ -8,19 +8,22 @@ public struct BuildAction: Equatable, Codable, Sendable {
     public var preActions: [ExecutionAction]
     public var postActions: [ExecutionAction]
     public var runPostActionsOnFailure: Bool
+    public var buildImplicitDependencies: Bool
```

Comment:
I am fine with both options actually. I chose buildImplicitDependencies because it matches the name of the property in the `xcscheme` file. So therefore I would leave it up to you, which one you prefer. The one shown on the UI or the one used in the file. 
<img width="400" alt="Screenshot 2024-10-21 at 18 28 59" src="https://github.com/user-attachments/assets/e2a6e769-65a2-4607-94cd-d87ec97f169a">

Reply:
I'd lean to `findImplicitDependencies` as the action name in the UI is what's meant to be user-facing whereas the name of the variable in `.pbxproj` is something that should be an implementation detail.

Reply:
Sure, makes sense - I changed the property accordingly